test/cli/package_missing_data_test.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/anchore/syft/syft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPackageMissingNameAndVersion(t *testing.T) {
+	formats := syft.FormatIDs()
+	commonAssertions := []traitAssertion{
+		func(tb testing.TB, _, stderr string, _ int) {
+			tb.Helper()
+			assert.Contains(tb, stderr, "python-package-cataloger: missing package name, that is necessary for further metadata extraction, such as CPEs and pURLs")
+			assert.Contains(tb, stderr, "python-package-cataloger: missing package version, that is necessary for further metadata extraction, such as CPEs and pURLs")
+		},
+		assertSuccessfulReturnCode,
+	}
+
+	for _, o := range formats {
+		t.Run(fmt.Sprintf("format:%s", o), func(t *testing.T) {
+			// TODO: does it make sense to add empty metadata for all catalogers?
```

Comment:
Metatada here as in data fixtures, simulating empty description files, like PKG-INFO for python projects.